Updated the RenderListings function

Tokle89 opened the pull request:
Rewrote the function to use createElements, instead of forcing a string into the DOM.
The function now also uses the API to get the data, instead of using a Lorem Ipsum, but still uses the same structure as before.
The function also uses the createElements function to create the elements, instead of creating them manually.
The Api call still comes from a dummy API, but the function is ready to be used with the actual API.
Deleted the searchListings import as it was never used

Now that the cards have different img and text, the styling needs to be adjusted, so the cards look alike

Olekrr commented on src/js/templates/listings/renderListings.js:
```diff
@@ -1,38 +1,104 @@
-       </div>
-       `;
+  const { products } = await getListOfListings();
+  console.log(products);
```
We should generally avoid having console.logs in our final product, as frequent logging can impact performance or inadvertently expose sensitive information

Tokle89 replied:
i Will fix this now, also discovered a problem on the button link i will fix at the same time

Olekrr reviewed (changes requested):
Code looks good to me, but the console.log should be removed

Well done!

Tokle89 commented:
I have removed the console logs

changed the dummy api url to the noroff api url

created a parseDate function that converts the deadline date to readable format

Olekrr commented:
Great, merging this :)
